tidb-cloud/scale-tidb-cluster.md

```diff
@@ -109,16 +109,26 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
 
 ## Increase node size
 
-When a cluster is running, you cannot increase its node size. To make such changes, take either of the following methods:
+You can increase the node size for TiDB, TiKV, and TiFlash. Decreasing the node size is not supported.
 
-- Method 1: Increase the node size through backup and restore
+To increase the node size, take one of the following methods:
+
+- Method 1: Increase the node size in the TiDB Cloud console
+
+    1. In the TiDB Cloud console, navigate to the **Clusters** page for your project.
+    2. Locate the cluster that you want to scale up, and click **...** in the upper-right corner of the cluster area.
+    3. Click **Modify** in the drop-down menu. The **Modify Cluster** page is displayed.
+    4. On the **Modify Cluster** page, increase the node size as you need.
+    5. Click **Confirm**.
+
+- Method 2: Increase the node size through backup and restore
```

Comment:
Are Method 2 (backup and restore) and Method 3 (support tickets) still necessary? 

IMO we can safely remove these methods now that users can scale up right in the console.

Reply:
+1